client/reader/start/main.jsx

```diff
@@ -55,8 +57,16 @@ const Start = React.createClass( {
 		page.redirect( '/' );
 	},
 
+	renderLoadingPlaceholders() {
+		const count = 4;
```

Comment:
You mean as a var name? I should probably go full verbose and do `numberOfPlaceholders` :)